src/Acuminator/Acuminator.Analyzers/DiagnosticsShortName.resx

```diff
@@ -372,4 +372,13 @@
   <data name="PX1056IsActiveForGraphMethod" xml:space="preserve">
     <value>PXGraphCreationInIsActiveForGraphMethod</value>
   </data>
+  <data name="PX1062" xml:space="preserve">
+    <value>StaticPropertyOrFieldInGraph</value>
+  </data>
+  <data name="PX1063" xml:space="preserve">
+    <value>ExceptionWithoutSerializationConstructor</value>
```

Comment:
@EAndrosova  please review diagnostic short messages for PX1063 and PX1064

Reply:
approved

Reply:
approved